src/Integration.Vsix.UnitTests/SonarLintTagger/AccumulatingIssueConsumerTests.cs

```diff
@@ -35,21 +36,26 @@ public class AccumulatingIssueConsumerTests
     {
         private static readonly IAnalysisIssue ValidIssue = CreateIssue(startLine: 1, endline: 1);
         private static readonly ITextSnapshot ValidTextSnapshot = CreateSnapshot(lineCount: 10);
+        private static readonly IIssueToIssueMarkerConverter ValidConverter = Mock.Of<IIssueToIssueMarkerConverter>();
```

Comment:
Is there a reason to call it Valid? Seems a bit confusing, as if it has a setup logic. Can be just `ConverterMock`.

Reply:
It's not typed as a mock so it should have "Mock" in the name. I use "Valid" as a prefix for reusable instances where the value isn't important and we don't need to do any checks against the instance.